### PR TITLE
add local dotnet tools

### PR DESCRIPTION
**Pull request by jrr**
This PR adds fsharplint, fantomas, and powershell, which can be invoked like:

```
jrr@jrrmbp ~/r/e/fsharp (feature/dotnet-tools)> dotnet tool restore
Tool 'dotnet-fsharplint' (version '0.12.3') was restored. Available commands: dotnet-fsharplint
Tool 'fantomas-tool' (version '2.9.2') was restored. Available commands: fantomas
Tool 'powershell' (version '6.2.3') was restored. Available commands: pwsh

Restore was successful.
jrr@jrrmbp ~/r/e/fsharp (feature/dotnet-tools)> dotnet fsharplint -sf generators/Track.fs
========== Linting generators/Track.fs ==========
========== Finished: 0 warnings ==========
========== Summary: 0 warnings ==========
jrr@jrrmbp ~/r/e/fsharp (feature/dotnet-tools)> dotnet fantomas generators/Track.fs
generators/Track.fs has been written.
jrr@jrrmbp ~/r/e/fsharp (feature/dotnet-tools)> dotnet pwsh ./test.ps1
Linting config.json
-> An implementation for 'bracket-push' was found, but config.json does not reference this exercise.
-> The implementation for 'bracket-push' is missing a README.
-> The implementation for 'bracket-push' is missing an example solution.
-> The implementation for 'bracket-push' is missing a test suite.
jrr@jrrmbp ~/r/e/fsharp (feature/dotnet-tools) [1]>
```

These seem like tools that contributors on this repo would want around, and providing them takes small steps toward https://github.com/exercism/fsharp/issues/306 and https://github.com/exercism/fsharp/issues/668 .

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
I love this, especially the addition of PowerShell as a global tool! Maybe we should add add section to the README.md file too (which is far too minimal by the way :( )?

**Comment by TeaDrivenDev** [comment]
It might - in theory - be nice to include BenchmarkDotNet here as well, as the way I've enabled running the benchmarks so far feels a bit kludgy. I say "in theory", because so far I haven't been able to actually run them through the tool with xUnit in the same project.

**Comment by jrr** [comment]
Readme updated.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,33 @@ We have a [gitter channel](https://gitter.im/exercism/xfsharp) where you can get
 
 Please see the [contributing guide](https://github.com/exercism/docs/tree/master/contributing-to-language-tracks)
 
+## Local Tools
+
+[PowerShell](https://github.com/PowerShell/PowerShell), [Fantomas](https://github.com/fsprojects/fantomas), and [FSharpLint](https://github.com/fsprojects/FSharpLint) are are available in this repo as [local tools](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#local-tools). Example usage:
```
Comment (ErikSchierboom) [inline README.md]: Maybe be explicit about the local tools requiring .NET Core 3.0?

**Comment by ErikSchierboom** [comment]
> I say "in theory", because so far I haven't been able to actually run them through the tool with xUnit in the same project.

Hmmm, weird. What is the error that you get?

File: README.md (hunk 1 above)
Comment (jrr) [reply README.md]: Updated.

**Comment by ErikSchierboom** [comment]
Merged. Thanks a lot @jrr!  🎉

**Pull request merged**